src/mixins/nft.js

```diff
@@ -331,6 +331,9 @@ export default {
         if (!result || !result.length) return undefined;
 
         const [price, ...prices] = result;
+        if (this.priceIndex !== undefined) {
+          return result[this.priceIndex].price;
```

Comment:
Should find by key "index" since there might be ordering, array index should only be used as fallback